### PR TITLE
SONARPY-1277 Add quick fix for S108 (EmptyNestedBlockCheck) in case of inline statements

### PR DESCRIPTION
**Pull request by maksim-grebeniuk-sonarsource**

### CODE REVIEW

File: python-frontend/src/main/java/org/sonar/python/tree/TreeUtils.java [hunk 1]
```diff
@@ -248,6 +248,41 @@ public static List<FunctionDef> topLevelFunctionDefs(ClassDef classDef) {
     return visitor.functionDefs;
   }
 
+  public static int findIndent(Tree tree) {
```
Comment (andreaguarino) [inline python-frontend/src/main/java/org/sonar/python/tree/TreeUtils.java:251]: I would suggest to rename it to `findIndentationSize`

File: python-frontend/src/test/java/org/sonar/python/tree/TreeUtilsTest.java [hunk 2]
```diff
@@ -441,6 +441,49 @@ public void test_getTreeByPositionComparator() {
...
+    assertThat(indent).isZero();
+  }
+
+  @Test
+  public void test_findIndentEmpty() {
+    FileInput fileInput = PythonTestUtils.parse("");
+    var indent = TreeUtils.findIndent(fileInput);
+    assertThat(indent).isZero();
+  }
+
```
Comment (andreaguarino) [inline python-frontend/src/test/java/org/sonar/python/tree/TreeUtilsTest.java:486]: Would be nice adding an example with inconsistent indentation, for example in same cases it's 2 and in other cases it's 4

File: python-checks/src/main/java/org/sonar/python/checks/EmptyNestedBlockCheck.java [hunk 3]
```diff
@@ -70,11 +70,21 @@ public void initialize(Context context) {
...
+          if (indent > 0) {
+            var offset = parent.firstToken().column() + indent;
+            var textToInsert = "\n" + " ".repeat(offset) + TODO_COMMENT_TEXT + "\n" + " ".repeat(offset);
+            var quickFix = PythonQuickFix.newQuickFix(QUICK_FIX_MESSAGE,
+              PythonTextEdit.insertBefore(passTreeElement, textToInsert));
+            issue.addQuickFix(quickFix);
+          }
         }
 
+
```
Comment (andreaguarino) [inline python-checks/src/main/java/org/sonar/python/checks/EmptyNestedBlockCheck.java:87]: Unnecessary change?

**Review by andreaguarino — APPROVED** [review]
LGTM, please have a look at my comments before merging

File: python-frontend/src/test/java/org/sonar/python/tree/TreeUtilsTest.java (hunk 2 above)
Comment (maksim-grebeniuk-sonarsource) [reply python-frontend/src/test/java/org/sonar/python/tree/TreeUtilsTest.java:584]: done

File: python-frontend/src/main/java/org/sonar/python/tree/TreeUtils.java (hunk 1 above)
Comment (maksim-grebeniuk-sonarsource) [reply python-frontend/src/main/java/org/sonar/python/tree/TreeUtils.java]: done

File: python-checks/src/main/java/org/sonar/python/checks/EmptyNestedBlockCheck.java (hunk 3 above)
Comment (maksim-grebeniuk-sonarsource) [reply python-checks/src/main/java/org/sonar/python/checks/EmptyNestedBlockCheck.java]: removed
